src/ui-strings.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { addStrings } from '@wordpress/components';
+import { select } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+export const uiStrings = () => {
+	const { getSettings } = select( 'core/block-editor' );
+	const unsupportedBlockEditor = {
+		supported:
+			getSettings( 'capabilities' ).unsupportedBlockEditor === true,
```

Comment:
Just asking out of curiosity. We have a lot of explicit bool checks in this file (i.e., `someValue === true` and `someValue === false`). Are we doing that to guard against a cases where `someValue` could be some other truth-y or false-y values, or is it a style preference to do the explicit bool check instead of doing something like `someValue` (or even `!!someValue` if we needed a bool) and `!someValue`, or maybe there's another reason?

Just asking because I tend to use the `!`s when I can (I find them more readable), and I'm curious if there are considerations I may not be considering when I do that. I'm not asking because I think we need to change it here. 🙂